src/OSPSuite.Infrastructure.Import/Core/DataFormat/MixColumnsDataFormat.cs

```diff
@@ -53,7 +53,7 @@ protected override UnitDescription ExtractUnits(string description, IUnformatted
          var unitKey = data.GetHeaders().FindHeader(description + "_UNIT");
          if (unitKey == null)
          {
-            return new UnitDescription();
+            return new UnitDescription(""); //in this case return the default unit for Dimensionless
```

Comment:
is this OK here? Or should I actually be getting the default unit for dimensionless from the dimensionFactory? I was thinking since the unit in the header is empty, it seems more consistent to leave it like this - maybe dimensionless is not only available in the dimensions in the end, and it ends up being of dimension fraction instead.